backend/services/interfaces/testService.ts

```diff
@@ -7,21 +7,27 @@ import {
   ShortAnswerMetadata,
   QuestionComponent,
 } from "../../models/test.model";
-import { UserDTO } from "../../types";
+import { UserDTO, AssessmentStatus, AssessmentType } from "../../types";
 
 export type TestResponseDTO = {
   /** the unique identifier of the response */
   id: string;
   /** the name of the test */
   name: string;
-  /** the duration of the test */
-  duration: number;
   /** the UserDTO for the admin */
   admin: UserDTO;
   /** an array of questions on the test */
   questions: QuestionComponent[][];
   /** the grade of the student */
   grade: number;
+  /** the type of assessment */
```

Comment:
I didn't actually know what the fields mean so these comments might not be that descriptive